tests/test_bs.py

```diff
@@ -296,6 +328,34 @@ def fin():
     return _setup_clients_in_namespace(testconfig['namespace'])
 
 
+@pytest.fixture(scope='module')
+def add_client(request, setup_oracle, setup_poet, setup_bootstrap):
+
+    def _add_single_client():
+        bs_info = setup_bootstrap.pods[0]
```

Comment:
if no bootstrap this will fail, add a message for that

Reply:
Done. Added check before.